doc/debugger.rst

```diff
@@ -1388,6 +1402,27 @@ reestablish the link).
 ``msg`` is an optional string elaborating on the reason for the detach.  It may
 or may not be present depending on the nature of detachment.
 
+AppNotify notification (0x07)
+-----------------------------
+
+Format::
+
+    NFY <int: 0x07> [<tval>]* EOM
```

Comment:
Ok, that makes sense. The text currently in is OK, just noticed there's some variability in the document now.